daemon/info.go

```diff
@@ -179,7 +179,11 @@ func (daemon *Daemon) fillPluginsInfo(v *types.Info) {
 func (daemon *Daemon) fillSecurityOptions(v *types.Info, sysInfo *sysinfo.SysInfo) {
 	var securityOptions []string
 	if sysInfo.AppArmor {
-		securityOptions = append(securityOptions, "name=apparmor")
+		profile := daemon.appArmorProfilePath
+		if profile == "" {
+			profile = "default"
+		}
+		securityOptions = append(securityOptions, fmt.Sprintf("name=apparmor,profile=%s", profile))
```

Comment:
Should this be profile-path or something similar instead of profile?

Reply:
I copied this from the seccomp profile for consistency (you're quite right that arguably they should both refer to "profile path" rather than "profile" but I don't know if changing both will break anything).

Reply:
sgtm